docs/2_4_common_schema.adoc

```diff
@@ -1743,9 +1743,9 @@ The following <<dependenciesKind>> is only allowed for floating point latexmath:
 
 ==== Terminals and Icons [[fmiTerminalsAndIcons]]
 
-Terminals define semantic groups of variables to ease connecting compatible signals on the system level.
+Terminals define semantic groups of variables to ease connecting compatible variables on the system level.
 This definition adds an additional layer to the interface description of the FMUs.
-It does not change the <<causality>> of the variables (i.e. inputs and outputs), but enables the definition of physical and bus-like  connectors that require special handling on the system level by the importer (e.g. flow variables and bus frames).
+It does not change the <<causality>> of the variables (e.g. inputs will remain inputs, and outputs will remain outputs), but enables the definition of physical and bus-like  connectors that require special handling on the system level by the importer (e.g. flow variables and bus frames).
```

Comment:
Isn't this also the place to clarify to readers that variables that are parameters can also be part of a terminal? (adding it to the e.g.)